darwin/cli.py

```diff
@@ -72,7 +72,7 @@ def run(args, parser):
         elif args.action == "export":
             f.export_dataset(args.dataset, args.include_url_token, args.annotation_class, args.name)
         elif args.action == "files":
-            f.list_files(args.dataset, args.status, args.path, args.only_filenames)
+            f.list_files(args.dataset, args.status, args.path, args.only_filenames, args.sort, args.direction)
```

Comment:
Being a CLI argument, I would call it `sort_by`. This should make the command a bit more readable
```
darwin dataset files v7/test-dataset complete --sort-by updated_at --direction desc
```

Reply:
Will update accordingly.